GitUI/UserControls/RevisionGrid/RevisionDataGridView.cs

```diff
@@ -51,7 +51,8 @@ public sealed partial class RevisionDataGridView : DataGridView
         private Font _monospaceFont;
 
         public bool UpdatingVisibleRows { get; private set; }
-        public bool IsBackgroundUpdaterActive => _backgroundUpdater.IsExecuting;
+        public bool IsGridUpdating => _backgroundUpdater.IsExecuting || IsLoadingGrid;
+        public bool IsLoadingGrid { get; set; } = false;
```

Comment:
I'm not finding the names of these two properties meaningful or consistent. In the context of this type we seem to use "revisions" not "grid" term.

Also this property feels like an implementation detail. In Windows Forms we use `BeginXyz`/`EndXyz` methods pair for such situations.

Reply:
There is a difference to Revisions (the "Git" data) and Grid (displayed rows). Grid is loaded after Revisions, not the same, so the name should differ.

Both of these could be replaced with a notification when Grid is loaded including the last selected row, but it just got complex when I tried this.

Reply:
Note that these methods are only used by tests

Reply:
We should avoid having "test only" API unless those are exposed via `TestAccessor`.

Reply:
IsGridUpdating  can be moved to test accessor, but how would you handle `IsLoadingGrid`?

Reply:
I pushed an alternative, please let me know what you think of it.